### DISCUSSION TITLE
write java copde

### DISCUSSION THREAD

**Discussion by lakeebhai** [Snailclimb/JavaGuide#2080] [General]
how to write java code but i am learn java completely but no practice and no skill please reply me

**Comment by lakeebhai** [comment]
please help me

**Comment by A1iMansour** [comment]
Once you understand the syntax things will be easy. You don't need any pre-knowledge. I would recommend that  you to go through "practice it "exercises, the best way to learn is to really write codes and fall into mistakes.
